### PR TITLE
Add WidowX 250 6DOF tutorial

### PR DESCRIPTION
**Pull request by jasiex01**
This tutorial overwrites the old WidowX 250 5DOF tutorial. It offers support for both versions, however the 6 DOF version is primary.

### CODE REVIEW

**Review by Krzemien97 — APPROVED** [review]
LGTM

File: docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx [hunk 1]
```diff
@@ -188,16 +206,14 @@ point to your result space. Open the file in `nano`:
 sudo nano /etc/ros/setup.bash
 ```
 
-Comment out the first line by adding `#` sign and add the source command for
-your workspace. The first 2 lines should look essentially like this:
+Add below line to this file:
```
Comment (bjsowa) [inline docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx]: Why did you change this line?

File: docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx [hunk 2]
```diff
@@ -263,100 +278,78 @@ the other nodes. You can manually restart the running nodes by typing:
...
-
-```bash
-sudo apt install python-catkin-tools
-sudo -H pip install modern_robotics
-```
+:::warning
 
-:::note
+Be careful while commanding the arm to move. It doesn't have collision
+detection.
```
Comment (bjsowa) [inline docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx]:
```suggestion
Be careful while commanding individual arm joints. The driver doesn't prevent
self-collisions in this mode of operation.
```

File: docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx
Comment (bjsowa) [inline docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx:1]: Replace `sudo nano` with just `nano`. There is no need for root privileges to edit files under `/etc/ros` in LeoOS.

Comment (bjsowa) [inline docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx:1]: On line 86, remove the  `(**addons** section)`. There is no such section.

**Review by bjsowa — CHANGES_REQUESTED** [review]

File: docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx (hunk 1 above)
Comment (jasiex01) [reply docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx]: Must have omitted this while copying, I'll revert it so it's more similar to original tutorial

**Comment by jasiex01** [comment]
All requested changes applied

**Review by bjsowa — APPROVED** [review]
